src/test/resources/RepoConfigurationTest/repoconfig_uniqueIdentifier_test2/local_alpha/test1.txt

```diff
@@ -0,0 +1 @@
+ hello
```

Comment:
What are this 2 files for? 
If its meant to allow the addition of empty folders, please do the gitignore solution in https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository

Do state the reason in the comments.

Reply:
Yes, it was to allow for the addition of empty folders. I have used the gitignore solution.